### ISSUE TITLE
Remove object from the array doesn't reload the view

### ISSUE THREAD

**Issue by Salarsoleimani** [gh123man/SwiftUI-LazyPager#5]
After removing one object from the data array. it is still showing on the page. could you please help me to find out the solution? 
Thanks

**Comment by gh123man** [comment]
Thanks for the report. I can confirm there are some issues with updating the underlying data, and direct indexing. 
I'll work on a fix and update this issue when it's ready.

**Comment by gh123man** [comment]
Can you try https://github.com/gh123man/SwiftUI-LazyPager/releases/tag/1.0.5 and let me know if it fixes your problem?

**Comment by Salarsoleimani** [comment]
the reloading works but the index instance doesn't update

**Comment by gh123man** [comment]
I would suggest updating the index manually to point to the correct element when the collection is updated. This library does not attempt to uniquely identify any data in the collection, so it has no way to know which index in the collection was removed/changed/reordered.

[I added a sample case for this in the example app here](https://github.com/gh123man/SwiftUI-LazyPager/blob/master/Examples/LazyPagerExampleApp/ContentView.swift#L88). 

The only time this library updates the binding index is when you intentionally change pages (+1 or -1). 

I need to update this library to make elements conform to `identifiable` so such a feature can be implemented correctly. This is something I am thinking about, and hope to implement soon.
